ios/RNCAsyncStorage.m

```diff
@@ -156,6 +162,52 @@ static dispatch_queue_t RCTGetMethodQueue()
   return error ? RCTMakeError(@"Failed to delete storage directory.", error, nil) : nil;
 }
 
+
+NSString *OldStorageDirectory = @"RNCAsyncLocalStorage_V1";
+/**
+ * Creates an NSException used during Storage Directory Migration.
+ */
+static NSException *RCTStorageDirectionMigrationException(NSString *reason, NSError *error) {
+  NSMutableString *reasonString = [[NSMutableString alloc] initWithString:reason];
```

Comment:
Instead of mutating a string, we can use `stringWithFormat:`:
```objc
NSString* detailedReason = [NSString stringWithFormat:@"%@: %@", reason, error.description];
```

Reply:
Ah thanks, I was looking for something like this. 😄